gatsby-config.js

```diff
@@ -2,7 +2,14 @@ module.exports = {
   pathPrefix: '/grad.then',
   siteMetadata: {
     title: 'grad.then()',
-    pages: ['jobs', 'learn', 'events', 'support', 'stories']
+    pages: ['jobs', 'learn', 'events', 'support', 'stories'],
```

Comment:
What's this part do? 🙃

Reply:
I think you've answered ☝️  this, here:

> * "static piece" of state is moved to gatsby-config.js and then is injected to Containers via graphQl queries on pages, 😉

Reply:
All 5 apps pages except **Main** have this `Filter&Search` component:
![image](https://user-images.githubusercontent.com/25195616/34725718-721b1312-f55a-11e7-9ffb-607fffb5e1a3.png)
The `Fiter` menu values previously lived in a state of a parent container. I moved all of them to `gatsby-config.js`.
I used the idea of @miljan-fsd , who made a separate file to store such values. :+1: